go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/metalmatze/signal v0.0.0-20210307161603-1c9aa721a97a
 	github.com/mitchellh/mapstructure v1.4.1
+	github.com/mwitkow/grpc-proxy v0.0.0-20181017164139-0f1106ef9c76
```

Comment:
is this a safe to use? Last commit is from October 2018

Reply:
@pavolloffay I had the same fear.  Reasons to keep it for now:
- I saw commits there from @bwplotka , so probably approves of mwitkow's approach
- I couldn't find anything else on github which forwards without deserializing and reserializing
- Because I have read mwitkow's code I can't recreate it myself (for legal reasons; anything I make might be seen as derivative to mwitkow's work).

Reply:
Yes, it seems there are not many alternatives (short of creating our own proxy implementation). I also found https://github.com/bradleyjkemp/grpc-tools which seem to be maintained a bit better, but I have no idea how it compares to this package.